.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
+  - todo
```

Comment:
I think that if there is a TODO in code, it should be for something that must be done shortly, so a warning for me is a good remainder. If it's not something that should be done shortly, then we should not add a todo and add a card in trello, and maybe just a comment explaining something, but not a todo.